frontend/src/container/IngestionSettings/IngestionSettings.tsx

```diff
@@ -0,0 +1,80 @@
+import './IngestionSettings.styles.scss';
+
+import { Table, Typography } from 'antd';
+import type { ColumnsType } from 'antd/es/table';
+import getIngestionData from 'api/settings/getIngestionData';
+import { useQuery } from 'react-query';
+
+interface DataType {
+	key: string;
+	name: string;
+	value: string;
+}
+
+export default function IngestionSettings(): JSX.Element {
+	const { data: ingestionData } = useQuery({
+		queryFn: () => getIngestionData(),
+	});
+
+	const columns: ColumnsType<DataType> = [
```

Comment:
Can we move this above `IngestionSettings` functions, or maybe shift this to a separate file, as it will be created every time the component re-renders?

Reply:
The data needs to be updated on rerender based on the response. The only possible reasons for re-render are the data or the parent rerendering